pkg/ssh/utils.go

```diff
@@ -44,50 +49,47 @@ func generateKeys() (pub, key []byte, err error) {
 	return publicKeyBytes, privateKeyBytes, nil
 }
 
-// generatePrivateKey creates a RSA Private Key of specified byte size
-func generatePrivateKey(bitSize int) (*rsa.PrivateKey, error) {
+// generatePrivateKey creates an ECDSA Private Key of specified byte size
+func generatePrivateKey(c elliptic.Curve) (*ecdsa.PrivateKey, error) {
 	// create key
-	privateKey, err := rsa.GenerateKey(rand.Reader, bitSize)
-	if err != nil {
-		return nil, err
-	}
-
-	// validate key
-	err = privateKey.Validate()
```

Comment:
is this `validate()` important to keep?

Reply:
The EC key doesn't have it. I don't think it is relevant either way, except if we expect the crypto library to create invalid keys